mosh/plan.sh

```diff
@@ -0,0 +1,25 @@
+pkg_name=mosh
+pkg_origin=core
+pkg_version=1.2.5
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('OCB-An-Authenticated-Encryption-Scheme')
```

Comment:
Seems that this [license](https://github.com/mobile-shell/mosh/blob/master/ocb-license.html) for Mosh is not listed in the SPDX [listing](http://spdx.org/licenses/).